requirements.txt

```diff
@@ -6,5 +6,6 @@ numpy
 properscoring
 scikit-learn
 scipy
+toolz
```

Comment:
Does toolz need to added in other .yaml/req.txt files?

Reply:
No it just needs to be added to the main `requirements.txt`. We discovered this same issue at `climpred`. If you were to have an absolute base python environment and try to install `python setup.py` or `pip install .`, it won't work without `toolz`. In having `xskillscore` as a dependency in the other yaml files, it will install `toolz`.